cmd/kubeadm/app/util/version.go

```diff
@@ -62,7 +65,13 @@ var (
 //  latest-1    (latest release in 1.x, including alpha/beta)
 //  latest-1.0  (and similarly 1.1, 1.2, 1.3, ...)
 func KubernetesReleaseVersion(version string) (string, error) {
-	return kubernetesReleaseVersion(version, fetchFromURL)
```

Comment:
kubernetesReleaseVersion is already supposed to be a helper method that eliminates internet calls in testing... if it is slow in tests, that seems like a bug to fix, not something we should work around with a cache